### DISCUSSION TITLE
How to make iterative search?

### DISCUSSION THREAD

**Discussion by DenisSvirin** [MilesCranmer/PySR#272] [Q&A] [answered]
Hello,
I need a model to search for equations on dataset1, then change parameters of this model and and search again but on (a different)
dataset2.And repeat this.
But seems it doesn't change data, because when it starts fitting second dataset, it gives me the same losses.
Will it work like that? or I need two models and fit equations from one model to another? 

https://gist.github.com/DenisSvirin/f3ba66627234908a0edebfd9c78a51d7

**Comment by MilesCranmer** [comment, marked as answer]
Coincidentally just yesterday I pushed a fix for this exact use case: https://github.com/MilesCranmer/SymbolicRegression.jl/pull/177

Next PySR update will have it, or before then you could change `version.py` to the latest SymbolicRegression.jl version and run `python setup.py install` and then `pysr.install()`.

**Reply by MilesCranmer** [reply]
Fixed with https://github.com/MilesCranmer/PySR/pull/271
